test/spec/modules/pubmaticBidAdapter_spec.js

```diff
@@ -978,6 +1033,15 @@ describe('PubMatic adapter', function () {
         expect(data.imp[1]['video']['w']).to.equal(multipleMediaRequests[1].mediaTypes.video.playerSize[0]);
         expect(data.imp[1]['video']['h']).to.equal(multipleMediaRequests[1].mediaTypes.video.playerSize[1]);
       });
+
+      it('Request params check for native ad', function () {
+        let request = spec.buildRequests(nativeBidRequests);
+        let data = JSON.parse(request.data);
+        expect(data.imp[0].native).to.exist;
+        expect(data.imp[0].native['request']).to.exist;
+        expect(data.imp[0].tagid).to.equal('div-1');
+        expect(data.imp[0]['native']['request']).to.exist.and.to.be.an('string');
```

Comment:
we have not checked if native-request is formed as expected

Reply:
done